Use the variables defined by the KDE4 cmake files to link against kio and kdeui.

--- client/config/CMakeLists.txt.orig	2017-03-29 09:58:23 UTC
+++ client/config/CMakeLists.txt
@@ -12,7 +12,7 @@ kde4_add_plugin(kwin_crystal_config ${kwin_crystal_con
 
 
 
-target_link_libraries(kwin_crystal_config  kdeui kio ${KDE4_KDEUI_LIBS} ${QT_QTGUI_LIBRARY} ${QT_QT3SUPPORT_LIBRARY})
+target_link_libraries(kwin_crystal_config  ${KDE4_KIO_LIBS} ${KDE4_KDEUI_LIBS} ${QT_QTGUI_LIBRARY} ${QT_QT3SUPPORT_LIBRARY})
 
 install(TARGETS kwin_crystal_config  DESTINATION ${PLUGIN_INSTALL_DIR} )
 
